external/js/cothority/spec/helpers/global-this.ts

```diff
@@ -0,0 +1,5 @@
+
+beforeAll(() => {
+    // @ts-ignore
+    global.globalThis = global;
```

Comment:
Thanks for the fix! In addition I would just explain why this is needed and how it is used.